### ISSUE TITLE
Not working; can't find location

### ISSUE THREAD

**Issue by MaazSoMillennial** [onlineth/Spotlight-Desktop#10]
BUild no: 16299
It's not working since I did clean windows reinstall. Says something like can't locate images. Please make it work, I really liked this feature. 
THanks

**Comment by jamadail** [comment]
Same issue here.

![image](https://user-images.githubusercontent.com/35995560/35618642-7b2f3044-0673-11e8-861d-e0ef5bc750fa.png)

**Comment by onlineth** [comment]
Huh, I'm on build 16299 as well and it seems to be working on my system. There is a known bug that I haven't got around to fixing yet, https://github.com/onlineth/Spotlight-Desktop/issues/7. This is most likely the issue, can you try that workaround and see if that works?

If that doesn't fix the issue, can you take a look [here ](https://github.com/onlineth/Spotlight-Desktop/wiki/Spotlight-Known-Locations-in-Registry)and report back where you find the updated image?

**Comment by jamadail** [comment]
![image](https://user-images.githubusercontent.com/35995560/35631728-8d5dcde2-069c-11e8-829f-7afd22600cf5.png)

I don't seem to have the `LandscapeAssetPath`

**Comment by jamadail** [comment]
The subtree:

![image](https://user-images.githubusercontent.com/35995560/35631892-0b198c4e-069d-11e8-8fd2-b5a7d55dc6ea.png)

![image](https://user-images.githubusercontent.com/35995560/35631927-2162fa80-069d-11e8-9b80-26b4ca07f2ec.png)

**Comment by jamadail** [comment]
Trying to find the files refered in `contentID`, I find the following:

![image](https://user-images.githubusercontent.com/35995560/35632277-1d07db58-069e-11e8-8dce-b005cd5b7f94.png)

in `C:\Users\*user*\AppData\Local\Packages\Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy\LocalState\TargetedContentCache\v3\338387\`

**Comment by onlineth** [comment]
Ok, looks like it is then that other bug, copy the path found in `landscapeImage` and paste that value into the name `LandscapeAssetPath` (make a new String Value) in the S-1-5-21... node.

![](https://i.imgur.com/t9b8Ard.png)

Reboot and you should be good to go

**Comment by jamadail** [comment]
Done: 
![image](https://user-images.githubusercontent.com/35995560/35634008-80b9a0d8-06a2-11e8-8521-e7eefcd8c99b.png)

got the same error:
![image](https://user-images.githubusercontent.com/35995560/35634090-b5dd1006-06a2-11e8-9d1d-6f95fbf4aa69.png)

**Comment by onlineth** [comment]
Not to ask a dumb question, did you reboot? If anything the application would find that location.

**Comment by jamadail** [comment]
Yes I did reboot. Shouldn't I find a file with the `contentId ` name in this path? Because I only find the ones I mencioned in the other message.

**Comment by onlineth** [comment]
Nope, this application doesn't look at the `contentId` key. This is quite weird though, it should find at the very least the copied path you put in already. Let's try this,

Open cmd, change directory into `%AppData%/onlineth/Spotlight Desktop` and then run this command, `RegLookup.exe HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Authentication\LogonUI\Creative`

Assuming it does give similar output as the image displays on my system, follow what I did here,

![](https://i.imgur.com/8gTX5PL.png)

**Comment by jamadail** [comment]
Done:

![image](https://user-images.githubusercontent.com/35995560/35639971-c1238146-06b3-11e8-9593-3ff25351dc3e.png)

**Comment by jamadail** [comment]
I get an error in step 2 because my user has a space between two word:

```
'C:\Users\XPTO' is not recognized as an internal or external command,
operable program or batch file.
```

The path would be like `C:\Users\XPTO XPTA\`

**Comment by onlineth** [comment]
Ah, that's my bad then. Didn't use quotation marks on the command, I'll try and get a release out shortly, thank you for helping debug.

**Comment by jamadail** [comment]
Yes, please send it. Looking forward to, thanks.

**Comment by onlineth** [comment]
@jamadail A pre-release is out which should take those paths straight from the registry and shouldn't get cut off. Let me know if this fixes your issue,

https://github.com/onlineth/Spotlight-Desktop/releases/tag/2.0.0

**Comment by jamadail** [comment]
Just installed and for the current image it worked. Let me tried for a few days, I'll get back to you next week. Thanks.
